src/app/main/components/movies-view/movies-view.component.ts

```diff
@@ -0,0 +1,43 @@
+import { PaginationService } from 'src/app/main/services/pagination.service';
+import { Movie } from 'src/app/main/models/movie.model';
+import { ApiService } from 'src/app/shared/services/api.service';
+import { Component, OnInit } from '@angular/core';
+import { BehaviorSubject, switchMap, take } from 'rxjs';
+import { MoviesService } from 'src/app/main/services/movies.service';
+
+@Component({
+  selector: 'app-movies-view',
+  templateUrl: './movies-view.component.html',
+  styleUrls: ['./movies-view.component.scss'],
+})
+export class MoviesViewComponent implements OnInit {
+  movies = new BehaviorSubject<Movie[]>([]);
```

Comment:
Use "public" explicitly to improve code readability. Also fix in other files, please